docs/documentation/helpcenter.md

```diff
@@ -148,6 +152,32 @@ with modals, etc. should never go in the help center documentation.
 In such cases, you may be able to fix the problem by adding text in-app,
 where the user will see it as they are interacting with the feature.
 
+### Redirecting an existing article
+
+From time to time, there is a reason to rename articles in the help center or
+REST API documentation. This breaks incoming links, including from other
+branches of the repository that haven't been rebased, and more importantly
+from previous versions of Zulip.
```

Comment:
We should mention the Zulip blog as a possible source as well.

Reply:
I believe you meant that published blog posts would be a source for broken links and not something we would rename/move. I updated the text to include that information.